Automattic/jetpack
# VideoPress: Fill data when entering on edit details page

Make sure that we're filling the data when user enter in edit details page directly.

#### Changes proposed in this Pull Request:

* Fill data when user enter in edit details page.

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Start `VideoPress`.
* Navigate to `Media Library`.
* Open the attachment.
* Click on `Edit Details`.
* You should be redirect to edit details page, and data be filled after fetching.
